src/commands/dev/dev.mjs

```diff
@@ -494,10 +505,20 @@ const dev = async (options, command) => {
     startPollingForAPIAuthentication({ api, command, config, site, siteInfo })
   }
 
+  log(`${NETLIFYDEVWARN} Setting up local development server`)
```

Comment:
I think it needs a different message for serve. I can see people being confused that it doesn't watch the source or config files, for example.

Reply:
Any suggestions?

Reply:
Had a go at this in https://github.com/netlify/cli/pull/5338/commits/0bcc9a26965ca1c84525f17ad429c5607e0063dd.